src/components/settings/environments/index.jsx

```diff
@@ -1,13 +1,11 @@
 import React from 'react';
 import styles from './environments.scss';
-import Table from '../table';
-import Navigation from '../navigation';
+import Table from '../../table';
 
 const Environments = () => (
-  <div className={styles['page-wrapper']}>
+  <div className={styles.environments}>
```

Comment:
TODO: let's use the class name for main element the same as component class name